### DISCUSSION TITLE
Send a request with mockoon

### DISCUSSION THREAD

**Discussion by s3nu4** [mockoon/mockoon#814] [Help] [answered]
hey,

maybe a dumb question, but is it possible to send a request with mockoon (like in postmans pre-request script)?

What  i want to achieve (i know this is a very special case) is that i send a request to a mocked mockoon api and send the same request from mockoon to another API... Like a "proxy"

Cheers :)

**Comment by 255kb** [comment, marked as answer]
You can proxy with Mockoon but any route that is declared will not be proxied.
There is currently no other way to make calls with Mockoon.

**Comment by s3nu4** [comment]
Hey @255kb,

thank you! I think this is what i need ;)

Cheers!
